docs/docs/dev_docs/debugging/main.md

```diff
@@ -18,19 +18,25 @@ use dep::aztec::oracle::debug_log::{ debug_log };
 ```
 
 **Write log**
-Write `debug_log()` in the appropriate place in your contract. 
+Write `debug_log()` in the appropriate place in your contract.
 
 ```rust
 debug_log("here")
 ```
 
 **Start Sandbox in debug mode**
 
-Prepend the command to start the sandbox with `DEBUG=aztec` to log everything or `DEBUG=aztec:simulator:oracle` to only log your `debug_log()` statements.
+Prepend the command to start the sandbox with `DEBUG=aztec:*` to log everything or `DEBUG=aztec:simulator:oracle` to only log your `debug_log()` statements.
 
 ```bash
-cd ~./aztec && DEBUG=aztec docker-compose up
+cd ~./aztec && DEBUG=aztec:* docker-compose up
```

Comment:
Should we assume the user has the docker compose file installed, when most will just be running the getting started script?

Maybe we should also include an example for the npm sandbox flow